src/ft/mod.rs

```diff
@@ -399,15 +397,17 @@ impl FreeTypeRasterizer {
             let font_pattern = &fallback_font.pattern;
             match self.faces.get(&font_key) {
                 Some(face) => {
-                    let index = face.ft_face.get_char_index(glyph.c as usize);
+                    let index = face.ft_face.get_char_index(glyph.character as usize);
 
                     // We found something in a current face, so let's use it.
                     if index != 0 {
                         return Ok(font_key);
                     }
                 },
                 None => {
-                    if font_pattern.get_charset().map(|cs| cs.has_char(glyph.c)) != Some(true) {
+                    if font_pattern.get_charset().map(|cs| cs.has_char(glyph.character))
+                        != Some(true)
```

Comment:
Found a pretty good pattern to handle this in my selection PR the other day, I think this is far superior to both `.unwrap_or(false)` and `!= Some(true)`:

```suggestion
                    if font_pattern.get_charset().map_or(false, |cs| cs.has_char(glyph.character))
```

Reply:
Only works when you have `.map` already though unfortunately.